lib/FrameManager.js

```diff
@@ -407,6 +407,25 @@ class Frame {
   }
 
   /**
+  *@param {string} selector
+  *@param {!Array<string>} values
```

Comment:
1. there should be spaces between "*" and "@"
2. this should also have a @return annotation:

```js
/**
  ...
  * @return {!Promise<!Array<string>>}
 */
```